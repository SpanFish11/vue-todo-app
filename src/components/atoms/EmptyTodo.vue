<script setup></script>

<template>
  <ul class="list-group list-group-flush d-flex align-items-center">
    <li class="list-group-item">
      <span class="text-secondary">No Todo yet</span>
    </li>
  </ul>
</template>
