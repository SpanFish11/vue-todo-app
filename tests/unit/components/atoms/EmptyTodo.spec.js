import { beforeEach, describe, expect, it } from '@jest/globals'
import { shallowMount } from '@vue/test-utils'
import EmptyTodo from '@/components/atoms/EmptyTodo.vue'

describe('Given the EmptyTodo component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(EmptyTodo, {})
  })

  it('should render', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
