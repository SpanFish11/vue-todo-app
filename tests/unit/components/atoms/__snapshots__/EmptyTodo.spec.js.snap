// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the EmptyTodo component should render 1`] = `
<ul
  class="list-group list-group-flush d-flex align-items-center"
>
  <li
    class="list-group-item"
  >
    <span
      class="text-secondary"
    >
      No Todo yet
    </span>
  </li>
</ul>
`;
